spec/models/user_spec.rb

```diff
@@ -47,6 +47,35 @@
       expect(User.authenticate(user.email, "bad_password")).to be_nil
     end
 
+    it "takes the same amount of time to authenticate regardless of whether user exists" do
```

Comment:
Is running just one comparison enough to feel confident here? Or should we run it multiple times and make sure all of them remain within the expected delta?

Reply:
Good question. How many times do you think would be reasonable?

Reply:
I honestly don't know if it is even necessary. I was just thinking of the fuzz  testing I've seen done in Elm apps.

Reply:
If I run each 100 times I can bump the `be_within` delta from 0.001 to 0.0001 (* 100). That is nice, but it also adds 0.4 seconds to our test suite. I am torn.

Reply:
I think I am going to leave it alone for now to keep the test suite as fast as possible. These tests in their current form are enough to catch a regression to the old behavior. We can always revisit this if we find that the 1ms delta is not sufficient.